customer.py

```diff
@@ -0,0 +1,86 @@
+import requests
+from datetime import datetime
+
+class Customer:
```

Comment:
This class and the two others like it are repetitive.  They all have the same job, to interact with the API on behalf of `main.py`.  Instead I suggest combining them into a single API_Wrapper class.   This class would make requests to the API and return the data back to the main module.